src/main/java/com/google/sps/data/MockDatastoreManager.java

```diff
@@ -129,4 +141,40 @@ private Hint getHint(int idx) {
         }
         return res.build();
     }
+
+    /**
+    * Returns a random integer in the interval [l,r].
+    * @param l the left endpoint of the interval (inclusive).
+    * @param r the right endpoint of the interval (inclusive).
+    * @return a random integer in [l,r].
+    */
+    private int getRandomIntegerBetween(int l, int r) {
+        int res = l + (int)(Math.random()*(r-l+1));
+        if(res > r) res = r; // happens if Math.random() somehow gives exactly 1
+        return res;
+    }
+
+    private String getRandomWord() {
+        String[] words = {"absorption", "knowledge", "wear", "egg",
+                          "befall", "staking", "light", "muddled",
+                          "dynamic", "attempt", "accurate", "meal",
+                          "trap", "progress", "walk", "muscle"};
+        return words[getRandomIntegerBetween(0, words.length-1)];
+    }
+
+    /**
+    * Creates a game with a random id, title, difficulty, and stars.
+    * All other fields are left as default.
+    * @return a random Game.
+    */
+    private Game getRandomGame() {
+        Game.Builder res = new Game.Builder(IDGenerator.gen(), getRandomWord() + " " + getRandomWord());
+        int numStarVotes = getRandomIntegerBetween(0, 10);
```

Comment:
Isn't the star scale 1-5? Why is there 10 here?

Reply:
numStarVotes is the number of people who voted, not the number of stars.